python/identifiers.cpp

```diff
@@ -59,6 +59,15 @@ void register_identifiers(pybind11::module& m) {
             "Use GPU backend.")
         .value("multicore", arb::backend_kind::multicore,
             "Use multicore backend.");
+
+    pybind11::enum_<arb::binning_kind>(m, "binning_kind",
```

Comment:
I am wondering if we should remove the `_kind` postfix in Python:
```
binning_kind::regular   -> binning::regular
backend_kind::multicore -> backend::multicore
```
It is a bit less verbose.

Thoughts?

Reply:
Sure, why not. Will do.